### PR TITLE
Ihar Hrachyshka - What did you do in Mitaka

### PR DESCRIPTION
**Pull request by rbowen**

### CODE REVIEW

**Comment by rbowen** [comment]
Publishes an older podcast with Ohad - moves it to PodBean.
Publishes a new podcast with Ihar.
Fixes http/https problem on embedded podcast player.

File: source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+the next release, operators will be able to retain their Neutron API
+available while controllers are upgraded one by one. That's one thing
+that I was working on.
+
+There are other things. There was a lot of work done in Neutron in the
+scope of Quality of Service (QOS). The initial support for QOS in
+Neutron was merged in Liberty, but it's still quite limited, so we
+expand on what we had done in that release. Specifically, the initial
+support was just for OpenVSwitch, but in Mitaka we added support for
+Linuxbridge ml2 drive. We also added role-based access control for QOS
```
Comment (booxter) [inline source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md]: ml2 driver

File: source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+policies. We added support for so-called DSCP tags, which is a feature
+to prioritize traffic based on which port it came from.
+
+Another interesting thing that we finally tackled in Mitaka is MTU
+support. This is one of the huge pain pain points that were identified
```
Comment (booxter) [inline source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md]: pain pain

File: source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+by operators in the past, that Neutron, while being the networking
+solution for OpenStack, cannot actually properly handle MTUs - which
+stands for Minimum Transfer Unit. Neutron could not actually handle,
```
Comment (booxter) [inline source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md]: minimum -> maximum

File: source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+neither standard MTUs for ethernet, nor so-called jumbo frames, which
+is really bad. It means that instances did not have access to the full
+capabilities of the underlying physicaly infrastructure. That should
```
Comment (booxter) [inline source/blog/2016-04-15-what-did-you-do-in-mitaka-ihar-hrachyshka-talks-about-neutron.html.md]: physical

**Pull request merged**